examples/dump_replay.lua

```diff
@@ -81,6 +81,24 @@ if (buildmap:ne(map.buildable_data):sum() ~= 0) then
   print("Buildability map doesn't match!, replayer is bugged!")
   return
 end
+print("Writing out walkmap => /tmp/walkmap.pgm, " ..
+      "heightmap => /tmp/heightmap.pgm, buildmap => /tmp/buildmap.pgm")
+local wmf = io.open("/tmp/walkmap.pgm", 'w')
+local hmf = io.open("/tmp/heightmap.pgm", 'w')
+local bmf = io.open("/tmp/buildmap.pgm", 'w')
+wmf:write("P2 " .. walkmap:size(2) .. " " .. walkmap:size(1) .. " 1\n")
+bmf:write("P2 " .. buildmap:size(2) .. " " .. buildmap:size(1) .. " 1\n")
+hmf:write("P2 " .. heightmap:size(2) .. " " .. heightmap:size(1) .. " 6\n")
+for y = 1, walkmap:size(1) do
+  for x = 1, walkmap:size(2) do
+    wmf:write(walkmap[y][x] .. " ")
+    hmf:write(heightmap[y][x] .. " ")
+    bmf:write(buildmap[y][x] .. " ")
+  end
+  wmf:write("\n")
+  hmf:write("\n")
+  bmf:write("\n")
+end
```

Comment:
Nit: you could store the maps in sth like `saved.{walkmap,heightmap,weightmap}` and iterate over the dictionary to save yourself some repetitions.